### PR TITLE
Fixed 'Cannot read property 'package' of undefined'

### LINKED ISSUE #6: Parcel 1.8.1 Cannot read property 'package' of undefined
When i update to Parcel-bundle 1.8.1 and compile error:  

Cannot read property 'package' of undefined
    at Bundler.bundler.on (/Users/swift/dev/element-starter/node_modules/parcel-plugin-sw-precache/index.js:16:43)
    at emitOne (events.js:116:13)
    at Bundler.emit (events.js:211:7)
    at Bundler.bundle (/Users/swift/dev/element-starter/node_modules/parcel-bundler/src/Bundler.js:288:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

### PR DESCRIPTION
**Pull request by adriantoine**
Hi, that fixes #6, at least it works well for me.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = bundler => {
   const { minify, publicURL, outDir } = bundler.options
 
   bundler.on('bundled', () => {
-    const pkg = require(bundler.mainAsset.package.pkgfile)
+    const pkg = bundler.mainBundle.entryAsset.package
```
Comment (cyyyu) [inline index.js:16]: This approach does not work for me with parcel-bundler@1.6 in which `bundler.mainBundle` is `undefined`.
While it does work with parcel-bundler@1.8.1.
Any idea to make it work for both?

Comment (cyyyu) [reply index.js:16]: Also I think we should update `peerDependecies.parcel-bundler` to `>1.8.x`.

**Comment by cyyyu** [comment]
I'm going to merge this and do the rest fixes by my side.
Thanks for this pr~

**Comment by adriantoine** [comment]
Hi @cyyyu, sorry for the late reply, I don't know if there's a way to be compatible with both `1.8` and `1.6`. I think you should release a major version to make sure people don't automatically upgrade and enforce Parcel version with peerDependencies.

Thanks for the merge!

**Comment by cyyyu** [comment]
@adriantoine It's ok :) .
I think v0.5.1 works for both now.

**Pull request merged**